### PR TITLE
Bugfix/#743 incorrect dropdown menu

### PR DESCRIPTION
**Pull request by nvolodymyr**

### CODE REVIEW

File: src/app/component/layout/components/header/header.component.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { ModalService } from '../../../core/components/propose-cafe/_modal/modal.service';
 import { MatDialog } from '@angular/material';
 import { UserSettingComponent } from '../../../user/components/user-setting/user-setting.component';
-import {NavigationStart, Router} from '@angular/router';
+import { NavigationStart, Router } from '@angular/router';
 import { LocalStorageService } from '../../../../service/localstorage/local-storage.service';
 import { JwtService } from '../../../../service/jwt/jwt.service';
 import { UserService } from 'src/app/service/user/user.service';
```
Comment (mplesha) [inline src/app/component/layout/components/header/header.component.ts:8]: Put in order those long-doted paths

File: src/app/component/layout/components/header/header.component.ts [hunk 2]
```diff
@@ -12,10 +12,11 @@ import { filter } from 'rxjs/operators';
 import { LanguageService } from '../../../../i18n/language.service';
 import { Language } from '../../../../i18n/Language';
 import { SearchService } from '../../../../service/search/search.service';
-import { UserOwnAuthService } from '../../../../service/auth/user-own-auth.service';
+import { UserOwnAuthService} from '../../../../service/auth/user-own-auth.service';
 import { SignInComponent } from '../../../auth/components/sign-in/sign-in.component';
 import { SignUpComponent } from '../../../auth/components/sign-up/sign-up.component';
```
Comment (mplesha) [inline src/app/component/layout/components/header/header.component.ts:17]: Put in order those long-doted paths

**Comment by mplesha** [comment]
conflicted PR

**Comment by mplesha** [comment]
confilcts

File: src/app/component/layout/components/header/header.component.ts (hunk 1 above)
Comment (nvolodymyr) [reply src/app/component/layout/components/header/header.component.ts]: fixed

File: src/app/component/layout/components/header/header.component.ts (hunk 2 above)
Comment (nvolodymyr) [reply src/app/component/layout/components/header/header.component.ts]: fixed

**Pull request merged**